Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8
```

Comment:
Doesn't look like the app files are ever copied over.

Reply:
They're shared in a volume in the docker-compose file.